### DISCUSSION TITLE
你们这个是什么网站啊？！

### DISCUSSION THREAD

**Discussion by fewyiohgrtyt** [share121/inter-knot#443] [其他]
你们这是害人不浅啊你们这个网站！！
麻烦你们。真的太过分了，你们搞这个网站干什么
我儿子以太类型的考试都不过那个平均分哪，他现在初二，你叫我儿子怎么办啊，他还不到高中好不好！！！？
你们这是什么网站啊
你们害死我儿子了
谁是站主，快点出来你们站主，再不来我去报警了啊！！！
我跟你们说你们这一帮人啊，
一天到晚啊搞这些什么盗洞客啊绳匠啊会害死你们的！！！！！！
![Image_1695081697452](https://github.com/user-attachments/assets/df6954b1-42b6-46e6-9d0c-f3c4b05e7310)

**Comment by Xmug** [comment]
这位家长您先别激动，首先我们这个网站上的人要么是新都大学的高材生，要么有独门绝技，不会害您的孩子的。
不过初中生确实不太适合上绳网，毕竟绳网许多内容都是和空洞相关，初中生还是好好学习才是。

**Comment by LHGS-github** [comment]
人活着就行（）

**Comment by 117518** [comment]
怎么感觉这么熟悉？

**Comment by huashuideryu** [comment]
哈哈哈哈，太经典了

**Comment by wudijians123** [comment]
经典

**Comment by maijiaosesu** [comment]
这位网友啊，我有鱼鱼蒸，能vivo50吗

**Comment by Mnpolize** [comment]
家长您先别激动，那您这样吧，您先打开任意支付软件，选择任意支付方式，用上述支付方式转100e丁尼到如下账户：@1sennada
我们法厄同会派遣教学邦布，为您的孩子提供上门教学服务！
如您满意的话还请打个好评呢亲~

**Comment by Rope-team** [comment]
温馨提示：绳网是面向于光大绳匠的综合任务交友平台

**Comment by Phaeton9528** [comment]
经典永流传了属于是

**Comment by Elysia-Ego-093** [comment]
啊这，死去的回忆突然开始攻击我😅

**Comment by MIHLFY** [comment]
经典永流传啊

**Comment by siler-wolf** [comment]
真是典中典啊

**Comment by light814** [comment]
Σ(°Д°;

**Comment by mixue4646** [comment]
好家伙经典咏流传

**Comment by hh244** [comment]
( つ•̀ω•́)つ

**Comment by ProxyPhaeton** [comment]
这个委托我接了。十万菲林，准让你家孩子次次考试拿满分🧐

**Comment by yipaf** [comment]
是绳网哟～

**Comment by StarrySkyehk** [comment]
啊这 , 真是典中典啊
